release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -146,6 +146,10 @@ For more information, see xref:../updating/updating-cluster-prepare.adoc#updatin
 ==== Minimum required permissions for Microsoft Azure to install and delete an {product-title} cluster
 In {product-title} {product-version}, instead of using the built-in roles, you can now define your custom roles to include the minimum required permissions for Microsoft Azure to install and delete an {product-title} cluster. These permissions are available for installer-provisioned infrastructure and user-provisioned infrastructure.
 
+[id="ocp-4-13-updating-clusters-migrating-cluster-to-multi-arch"]
+==== Convenient single-architecture to multi-architecture payload migration
+{product-title} {product-version} introduces the `oc adm upgrade --to-multi-arch` command, which allows you to migrate your current cluster with single-architecture compute machines to a cluster with multi-architecture compute machines. By updating to a multi-architecture, manifest-listed payload, you can add mixed architecture compute nodes to your cluster.
```

Comment:
```suggestion
{product-title} {product-version} introduces the `oc adm upgrade --to-multi-arch` command, which lets you migrate a cluster with single-architecture compute machines to a cluster with multi-architecture compute machines. By updating to a multi-architecture, manifest-listed payload, you can add mixed architecture compute nodes to your cluster.
```

Reply:
Actually one more comment @obrown1205. I suggest keeping things parallel and go with either "compute machines" or "compute nodes".

Reply:
agreed :) thanks!